### ISSUE TITLE
Not allowing the same member to join the group multiple times?

### ISSUE THREAD

**Issue by Bren2010** [mlswg/mls-protocol#700]
In #675, it was pointed out that we previously had a clause:

> If there are multiple Add proposals containing KeyPackages that the committer considers to represent the same client or a client already in the group (for example, identical KeyPackages or KeyPackages sharing the same Credential), the committer again chooses one to include and considers the rest invalid.

But this was amplified in #675 to say that we also can't send Add proposals for users that are already in the group. I think we should talk about if this is the actual desired behavior or not.

**Comment by rohan-wire** [comment]
I agree with Brendan. Specifically I want to make sure an MLS client can replace itself. 

My other concern is that because we have no guidance yet on identity, we need to make sure that multiple instances/clients of the same user are allowed.

**Comment by bifurcation** [comment]
Interim 2022-05-26:

* We want to:
    * Avoid having multiple appearances of the same device
    * Allow a device to resync itself
* Text at issue here is "An Add proposal is invalid if ... The Credential in the KeyPackage represents a client already in the group according to the application" 
* It sounds like changing to "... The KeyPackage represents a client already in the group according to the application" would address the immediate concern
* @Bren2010 to write a PR
